pkg/scheduler/plugins/elasticquota/plugin_helper.go

```diff
@@ -26,7 +26,9 @@ import (
 	v1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
 	quotav1 "k8s.io/apiserver/pkg/quota/v1"
+	"k8s.io/gengo/examples/set-gen/sets"
```

Comment:
Use `k8s.io/apimachinery/pkg/util/sets` instead of `k8s.io/gengo/examples/set-gen/sets `.

Reply:
fixed